backend/api/quivr_api/modules/knowledge/entity/knowledge.py

```diff
@@ -68,3 +68,5 @@ class KnowledgeDB(SQLModel, table=True):
     metadata_: Optional[Dict[str, str]] = Field(
         default=None, sa_column=Column("metadata", JSON)
     )
+
+    knowledge_brain: List[KnowledgeBrain] = Relationship(back_populates="knowledge")
```

Comment:
That's not how many-to-many model work in sqlmodel. Take a look at :

https://sqlmodel.tiangolo.com/tutorial/many-to-many/create-data/